projects/demo/src/modules/components/range/index.html

```diff
@@ -111,20 +110,6 @@
                     means percentage of total track length.
                 </p>
             </tr>
-            <tr
-                name="[size]"
```

Comment:
Let's add `CSS customization` section instead of it.
Similar to this one 
https://taiga-ui.dev/components/progress-circle/API?style.--tui-thickness.px=8

<img width="757" height="178" alt="Image" src="https://github.com/user-attachments/assets/cf2f76c9-97da-47f6-9797-7ac35fb779fb" />